src/components/treeInfo/index.tsx

```diff
@@ -47,7 +47,7 @@ const TreeInfo: React.FC<TreeProps> = ({
   const history = useHistory();
   const location = useLocation<RedirectStateProps>();
 
-  const adopted = siteData.entries[0].adopter !== null;
+  const adopted = siteData.entries[0] && siteData.entries[0].adopter !== null;
```

Comment:
Going off of the optional chaining operator comment below, this line should actually be changed to `!!siteData.entries[0]?.adopter` because of how the optional chaining operator returns `undefined` if the object is `undefined` but I admit this isn't super readable so I think what we have now is good

Double exclamation mark: https://stackoverflow.com/questions/29312123/how-does-the-double-exclamation-work-in-javascript